contracts/Registry.sol

```diff
@@ -12,6 +12,26 @@ contract Registry is IRegistry {
     mapping(bytes32 => Entry) public modules;
     bytes32[] public moduleKeys;
 
+    ///@notice emitted when governance address is changed
+    ///@param newGovernance the new governance address
+    event governanceChanged(address newGovernance);
+
+    ///@notice emitted when a contract is added to registry
+    ///@param newContract address of the new contract
+    ///@param moduleId keccak of module name
+    event newContractEvent(address newContract, bytes32 moduleId);
```

Comment:
Can you use the same convention as the other events, and so not use the word `Event` here, to keep it consistent?